### DISCUSSION TITLE
Looking for a complete list of Modules or Apps to install

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/arozos#113] [Q&A] [answered]
Managed to install Arozos on Virtualbox, need to see where is a list of thins to install

**Comment by tobychui** [comment, marked as answer]
In shorts, you can find the list of WebApps here:
https://github.com/aroz-online/WebApp-and-Subservice-Index

Long story: 
Most of the apps that is free to use are packed into the main release. There are (little) open source apps developed for this system as this system is still in development and didn't attract that much developer interests. However, companies that use this system do have their  proprietary apps built to suit their own needs. I know this because there are a few emails coming in monthly that asking for technical details for integrations and supports. 

A side notes, GPLv3 license used by ArozOS only covers the system core, many of the WebApp developers tends to not open source their apps. I guess it is because most of them involve proprietary code for their system integrations and daily workflows. If you are planning to build your open source apps on this system, I can provide on demand free email technical & integration support 👍🏻

**Reply by Dvalin21** [reply]
Thanks for the heads up. I just started Java for Beginners in college so it may be awhile before I get to the level, however it would be pretty cool to try as a school protect later on. I do believe in my course path I have to take advance java anyway. Ill keep watch of ArozOS, I think its pretty awesome. Hope it goes the distance like I believe it can.
